Scripts/run_tests.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+readonly SCRIPT_ABS_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)"
+
+readonly PROJECT_ROOT_PATH="$SCRIPT_ABS_PATH/.."
+
+readonly PLATFORM_IOS="iOS Simulator,name=iPhone 14 Pro"
+readonly PLATFORM_TVOS="tvOS Simulator,name=Apple TV 4K (3rd generation) (at 1080p)"
+
+readonly XCODE_PROJECT="FingerprintJS.xcodeproj"
+readonly XCODE_SCHEME="FingerprintJS"
+
+cd "$PROJECT_ROOT_PATH"
+
+function xcodebuild_test() {
+    xcodebuild test \
+        -project Sources/$XCODE_PROJECT \
+        -scheme $XCODE_SCHEME \
```

Comment:
What you are referring to here is called *word splitting* and it is evaluated during variable expansion. The delimiter used by the word splitting is defined by a special variable called IFS (Internal Field Separator) and its default value is a combination of three characters: space, tab, and newline. Now, in our case neither `XCODE_PROJECT` nor `XCODE_SCHEME` contains any of these delimiters, so nothing unexpected will happen. However, it certainly won't hurt if we quote these expressions, so I will amend the PR. It will make the code more bulletproof, should the project or scheme name include whitespaces for whatever reason.

Reply:
Done.